packages/llamaindex/tests/vectorStores/SimpleVectorStore.test.ts

```diff
@@ -74,12 +74,54 @@ describe("SimpleVectorStore", () => {
             {
               key: "private",
               value: "false",
-              filterType: "ExactMatch",
+              operator: "==",
             },
           ],
         },
       });
       expect(result.similarities).length(2);
     });
+    it("able to query nodes with filter IN", async () => {
+      await store.add(nodes);
+      const result = await store.query({
+        queryEmbedding: [0.1, 0.2],
+        similarityTopK: 3,
+        mode: VectorStoreQueryMode.DEFAULT,
+        filters: {
+          filters: [
+            {
+              key: "dogId",
+              value: ["1", "3"],
+              operator: "in",
+            },
+          ],
+        },
+      });
+      expect(result.similarities).length(2);
+    });
+    it("able to query nodes with filter condition OR", async () => {
+      await store.add(nodes);
+      const result = await store.query({
+        queryEmbedding: [0.1, 0.2],
+        similarityTopK: 3,
+        mode: VectorStoreQueryMode.DEFAULT,
+        filters: {
+          filters: [
+            {
+              key: "private",
+              value: "false",
+              operator: "==",
+            },
+            {
+              key: "dogId",
+              value: ["1", "3"],
+              operator: "in",
+            },
+          ],
+          condition: "or",
```

Comment:
the two tests only differ in 
1. the filters array
2. the expected length

extract this data in a testdata array, e.g.:

```
const testcases = [
{
   title: "able to query nodes with filter condition OR"
  filters: {
          filters: [
            {
              key: "private",
              value: "false",
              operator: "==",
            },
            {
              key: "dogId",
              value: ["1", "3"],
              operator: "in",
            },
          ],
          condition: "or",
        },
    similarities: 3
}
```

and test also other conditions